website/docs/d/rds_engine_version.markdown

```diff
@@ -0,0 +1,48 @@
+---
+subcategory: "RDS"
+layout: "aws"
+page_title: "AWS: aws_rds_engine_version"
+description: |-
+  Information about an RDS engine version.
+---
+
+# Data Source: aws_rds_engine_version
+
+Information about an RDS engine version.
+
+## Example Usage
+
+```hcl
+data "aws_rds_engine_version" "test" {
+  engine             = "mysql"
+  preferred_versions = ["5.7.42", "5.7.19", "5.7.17"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `engine` - (Required) DB engine. Engine values include `aurora`, `aurora-mysql`, `aurora-postgresql`, `docdb`, `mariadb`, `mysql`, `neptune`, `oracle-ee`, `oracle-se`, `oracle-se1`, `oracle-se2`, `postgres`, `sqlserver-ee`, `sqlserver-ex`, `sqlserver-se`, and `sqlserver-web`.
+* `parameter_group_name` - (Optional) The name of a specific DB parameter group family. Examples of parameter group families are `mysql8.0`, `mariadb10.4`, and `postgres12`.
+* `preferred_versions` - (Optional) Ordered list of preferred engine versions. The first match in this list will be returned. If no preferred matches are found and the original search returned more than one result, an error is returned.
+* `version` - (Optional) Version of the DB engine. For example, `5.7.22`, `10.1.34`, and `12.3`.
```

Comment:
We should mention the behavior here that if both version arguments are unspecified, the data source will return the default version.

```suggestion
* `version` - (Optional) Version of the DB engine. For example, `5.7.22`, `10.1.34`, and `12.3`. If both the `version` and `preferred_versions` arguments are not configured, the data source will return the default version for the engine.
```